runtime/lua/vim/_inspector.lua

```diff
@@ -150,12 +150,12 @@ function vim.show_pos(bufnr, row, col, filter)
 
   ---@private
   local function append(str, hl)
-    table.insert(lines[#lines], { str, hl })
+    lines[#lines][#lines[#lines] + 1] = { str, hl }
```

Comment:
I don't see the value in this, tbh. This line shows how readability is greatly reduced, while there is probably an insignificant performance gain (if at all).

Reply:
The addition of 1 to the tail already indicates that this is a tail insertion. I don't think the readability is diminished

Reply:
This is an instance where the `table.insert` version is much easier to read. In this case, readability trumps performance because this is not performance sensitive code.